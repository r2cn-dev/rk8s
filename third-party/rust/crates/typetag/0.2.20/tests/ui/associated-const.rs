#[typetag::serde]
pub trait Trait {
    const ASSOC: u8;
}

fn main() {}
