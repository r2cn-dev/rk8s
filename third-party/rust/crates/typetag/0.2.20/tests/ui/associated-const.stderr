error: typetag trait with associated const is not supported yet
 --> tests/ui/associated-const.rs:3:5
  |
3 |     const ASSOC: u8;
  |     ^^^^^^^^^^^^^^^^
